app/src/test/java/org/lfenergy/compas/sitipe/rest/v2/BTComponentResourceTest.java

```diff
@@ -95,12 +95,13 @@ void itShouldThrowErrorWhenImportedComponentNotFound() {
         var response = given()
                 .when().get("/imported/{id}", 10)
                 .then()
-                .statusCode(500)
+                .statusCode(404)
                 .extract()
                 .response();
 
+
         JsonPath jsonPath = response.jsonPath();
 
-        assertEquals("CORE-9999", ((LinkedHashMap<String, ?>)((ArrayList)((LinkedHashMap<String, ?>)jsonPath.get()).get("errorMessages")).get(0)).get("code"));
+        assertEquals("Imported BT Component not found", jsonPath.getString("message"));
```

Comment:
It's better that this message is thrown, since the WebApplicationException.NotFound is thrown by the Service